backend/src/worker.rs

```diff
@@ -1626,25 +1626,36 @@ async fn handle_child(
     let output = child_joined_output_stream(&mut child);
     let job_id = job_id.clone();
 
+    let (tx, mut rx) = mpsc::channel::<()>(1);
```

Comment:
does broadcast not work, like for the killpill which this is essentially ?

Reply:
Also, let's give it a more explicit name: kill_tx, kill_rx

Reply:
Broadcast would also work, but I try to use the smallest feature set for channels, to allow optimizations by tokio. Changed this to use oneshot now.

Reply:
:+1:

Reply:
rename would still be appreciated